.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: olafurpg/setup-scala@v2
+        with:
+          java-version: 11
```

Comment:
The official `actions/setup-java` action configures scala also, doesn't it? I think it breaks on Mac, but works on Linux. Is there a reason to use this one instead of that one?